examples/api/lib/widgets/focus_scope/focus.1.dart

```diff
@@ -44,12 +44,19 @@ class FocusableText extends StatelessWidget {
         // would be the one given the FocusableText build function, and that
         // would start looking for a Focus widget ancestor of the FocusableText
         // instead of finding the one inside of its build function.
-        return Container(
-          padding: const EdgeInsets.all(8.0),
-          // Change the color based on whether or not this Container has focus.
-          color: Focus.of(context).hasPrimaryFocus ? Colors.red : Colors.white,
-          child: Text(data),
-        );
+        
+        // Wrap Container in GestureDetector to change focus on tap
+        return GestureDetector(
+            onTap: () {
+              Focus.of(context).requestFocus();
+            },
+            child: Container(
+              padding: const EdgeInsets.all(8.0),
+              // Change the color based on whether or not this Container has focus.
+              color:
+                  Focus.of(context).hasPrimaryFocus ? Colors.red : Colors.white,
+              child: Text(data),
+            ));
```

Comment:
```suggestion
        final FocusNode focusNode = Focus.of(context);

        return GestureDetector(
          // This focus node's requestFocus() method is called when the widget is tapped.
          onTap: focusNode.requestFocus,
          child: Container(
            padding: const EdgeInsets.all(8.0),
            // Change the color based on whether or not this Container has focus.
            color: focusNode.hasPrimaryFocus ? Colors.red : Colors.white,
            child: Text(data),
          ),
        );
```

Defining a `focusNode` variable this way would have it sit right beneath the explanation of its purpose, and it'd allow for a nice-looking tearoff :)

Reply:
Hmm. It never occurred to me to press the "tab" key. I guess should have thought about that as I'm trying to write an app and am frustrated that "tab" and "enter" don't seem to work the same way. (I could very well be doing something wrong.) Perhaps it would be good to update the web page to include how focus changes (automatically) with certain key presses and how that behavior differs from key to key. I agree with your suggested change. How does this move forward?

Reply:
> Perhaps it would be good to update the web page to include how focus changes (automatically) with certain key presses and how that behavior differs from key to key.

Agreed—sounds like a great idea for a future contribution! Feel free to open another pull request for this, or if you'd rather just add it to our todo-list, you're welcome to [open an issue](https://github.com/flutter/flutter/issues/new/choose).

<br>

> I agree with your suggested change. How does this move forward?

Sweet! I'll hit the "commit suggestion" button. In the future, feel free to do that yourself when you see a suggestion you like :)